backend/backend/tests/test_database_check_view.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.test import APITestCase, APIClient
+from rest_framework.views import status
+from backend.views.database_check_view import DatabaseCheckModel
+from backend.serializers import database_check_serializer
+
+class DatabaseCheckViewTests(APITestCase):
+    client = APIClient()
+
+    @staticmethod
+    def create_database_entry(name=''):
+        if (name) != '':
+            DatabaseCheckModel.objects.create_database_entry(name=name)
+
+    def setUp(self):
+        self.create_database_entry(name='name1')
+
+class GetAllDatabaseEntriesTest(DatabaseCheckViewTests):

```

Comment:
All of these tests seem to be testing the model, not the `DatabaseViewSet` view itself. Should we move them to `test_database_check_model`? Does it make sense to have separate /tests/models and /tests/views folders?

Reply:
These tests are kind of testing both:

Here, we're testing the model itself by calling a method inside it:

![image](https://user-images.githubusercontent.com/23021242/52276592-9e1e7800-2920-11e9-91c1-b33ced7a14db.png)

However, here we're testing the view itselft by using the expected _"SELECT Query"_:

![image](https://user-images.githubusercontent.com/23021242/52276720-edfd3f00-2920-11e9-98b3-ea544de10ce5.png)

Maybe we could rename the file? Like _'test_database_check_api'_? Or _'test_database_check_view_and_model'_, but this is kind of a long name?
What do you think @caleybrock?

Reply:
Renamed the test file + added a comment.
See commit #https://github.com/code-for-canada/project-thundercat/pull/44/commits/069974658abead035b9fb02dd8dd68e3a1c62804